core/lcsb/rems.py

```diff
@@ -212,6 +228,26 @@ def create_rems_entitlement(obj: Union[Access, User],
     else:
         klass = obj.__class__.__name__
         raise TypeError(f'Wrong type of the object - should be Contact or User, is: {klass} instead')
-        
+
     new_logbook_entry.save()
     return True
+
+
+def build_access_notes_and_rems_user(application):
```

Comment:
Could you add some type hints to this method?

Reply:
I've took advantage for that to split into two functions. Didn't really made sense to have the note and the user together.